src/components/Code.tsx

```diff
@@ -195,7 +195,11 @@ const lightStyles = StyleSheet.create({
   'hljs-attr': {color: lightColors.text}, // Attributes within JSX Tags
 });
 
-function renderLowLightNode(node: lowlight.AST.Unist.Node) {
+function createKey(value: string) {
+  return `${value}_${new Date().getTime()}`;
```

Comment:
Won't this ensure that the keys are always different? (as each render will append the index to a new date)
That would result in always going down the slow path of throwing away the old UI and creating new UI.

Reply:
Ah I see. I removed the timestamp from the key and replaced it with the text value or element tagName depending on the type of the node.